common/app/implicits/Requests.scala

```diff
@@ -24,7 +24,7 @@ trait Requests {
 
     lazy val isAmp: Boolean = r.getQueryString("amp").isDefined || (!r.host.isEmpty && r.host == Configuration.amp.host)
 
-    lazy val isEmail: Boolean = r.getQueryString("format").contains("email") || r.path.endsWith(EMAIL_SUFFIX)
+    lazy val isEmail: Boolean = r.getQueryString("format").exists(_.contains("email")) || r.path.endsWith(EMAIL_SUFFIX)
```

Comment:
I'm assuming that Fastly will let `format=email-connected` through, since it looks to me like the parameter just has to *start* with `email`: https://github.com/guardian/fastly-edge-cache/blob/312b4728f0799f2677e48c249085c08faee08da2/theguardiancom/src/main/resources/varnish21/backend-nextgen.vcl#L79